python/iceberg/core/base_metastore_tables.py

```diff
@@ -16,28 +16,51 @@
 # under the License.
 
 from iceberg.api import Tables
-from iceberg.exceptions import NoSuchTableException
+from iceberg.exceptions import AlreadyExistsException, CommitFailedException, NoSuchTableException
 
 from .base_table import BaseTable
+from .table_metadata import TableMetadata
 
 
 class BaseMetastoreTables(Tables):
+    DOT = '.'
 
     def __init__(self, conf):
         self.conf = conf
 
     def new_table_ops(self, conf, database, table):
         raise RuntimeError("Abstract Implementation")
 
-    def load(self, database, table):
+    def load(self, table_identifier):
```

Comment:
Minor: if we add types, the docs are better. For example, changing this to `table_identifier: str` made it so that documentation for `str.rsplit` is found, at least in PyCharm. We should probably start adding types where it makes sense, like in the base class `Tables`.

Not something we need to do in this commit, but it really helps reviewing.